README.md

```diff
@@ -1 +1,197 @@
-# Math-magician
\ No newline at end of file
+<a name="math-magician-projects"></a>
+
+# THIERRY NOUMBUH (AXCELL EVANS)
+
+  <h1><b>math-magician-projects</b></h1>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# math-magician-projects
+This project is math-magician-projects microvers which will be used for learning in the microverse. No complex codind is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+1- react
+2- css
+3- linters
+4- javascript
+5-modules
+
+## 🛠 Built With <a name="built-with"></a>
+
+Visual Studia Code, HTML/JavaScript linters, ES6, Webpack, API.
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>JavaScript ES6</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+  <summary>REACT</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Linters implementation]**
+- **[To-Do List application]**
+- **[Single Page application]**
+- **[JavaScript Modules]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# LIVE DEMO 
+
+- [] (coming soon)
+
+
```

Comment:
- Optional
I would recommend removing this line from your README file, till you deploy this app.